integreat_cms/google_translate_api/google_translate_provider.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+import logging
+
+from django.apps import apps
+from django.conf import settings
+from django.utils.functional import classproperty
+
+from ..core.utils.machine_translation_provider import MachineTranslationProvider
+from .google_translate_api_client import GoogleTranslateApiClient
+
+logger = logging.getLogger(__name__)
+
+
+class GoogleTranslateProvider(MachineTranslationProvider):
+    """
+    The provider for Google machine translations
+    """
+
+    #: The readable name for this provider
+    name = "GoogleTranslate"
```

Comment:
There's intentionally no space between the words?

Reply:
Oh right, it's not a variable name or something like that. Yes, a space is added 😉